### PR TITLE
[V0.5.0] Release v0.5.0

### PR DESCRIPTION
**Pull request by jazz-cb**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+- Add `Coinbase.configure` method to allow for configuration of the SDK and marked constructor as deprecated.
```
Comment (rohan-agarwal-coinbase) [inline CHANGELOG.md:5]: we should add `## [0.5.0] - 2024-09-11` here right? unless you plan to in a separate PR

**Review by rohan-agarwal-coinbase — APPROVED** [review]
🔥 looks good!

**Comment by cb-heimdall** [comment]
Review Error for rohan-agarwal-coinbase @ 2024-09-11 22:07:13 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by rohan-agarwal-coinbase — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jazz-cb) [reply CHANGELOG.md:8]: added in #235

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-nodejs/issues/236) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-nodejs/issues/236) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-nodejs/issues/236) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-nodejs/issues/236) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-nodejs/issues/236) to review and +1

**Pull request merged**